frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskLogs.tsx

```diff
@@ -129,38 +132,59 @@ class PipelineTaskLogs extends React.Component<PipelineTaskLogsProps, PipelineTa
     const { obj, taskName } = this.props;
     const { renderContainers, fetching, targetHeight } = this.state;
     const containerStatus = _.get(this.props, ['obj', 'data', 'status', 'containerStatuses'], []);
+    const downloadLogs = () => {
```

Comment:
This function should moved to the class instead of created on render since this is a class component.

Reply:
Moved to a class member